checkers/forbidden_import_checker.py

```diff
@@ -1,26 +1,42 @@
+import astroid
 from pylint.interfaces import IAstroidChecker
 from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
 
 
 class ForbiddenImportChecker(BaseChecker):
     __implements__ = IAstroidChecker
 
     name = 'custom'
     msgs = {'E9999':
-                ('You may not import any modules - you imported \033[4;34m%s\033[0m on line %s.',
```

Comment:
Actually, I just changed the num of arguments in the message. In my checker, it simply detected the use of __import__ function.  And the message shows like this：
```
E9999: (forbidden-import)
    You may not import any modules - you imported re on line 1.
E9999: (forbidden-import)
    You may not import any modules - you use __import__ on line 2
```